src/devTools.js

```diff
@@ -138,6 +143,7 @@ function liftReducer(reducer, initialState) {
         currentStateIndex++;
       }
       stagedActions = [...stagedActions, liftedAction.action];
+      timestamps = [...timestamps, Date.now()];
       break;
```

Comment:
They wouldn't need timestamps. We only want timestamp alongside user's actions. Therefore they should be assigned in `liftAction()` and 

```js
       stagedActions = [...stagedActions, liftedAction.action];
      timestamps = [...timestamps, Date.now()];
```

would be 

```js
       stagedActions = [...stagedActions, liftedAction.action];
       timestamps = [...timestamps, liftedAction.timestamp];
```

Even if you put timestamps on COMMIT, etc, there is no code that would read them.